formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -2051,16 +2051,16 @@ and mutuallyRecursiveTwo y => print_int y;
 /* let newMutualRecursionSyntax x => newMutuallyRecursiveTwo (x + x); */
 /* let newMutuallyRecursiveTwo y => print_int y; */
 /*  */
-type x = private int;
+type x = pri int;
 
-type myType 'a 'b 'c = private ('a, 'b, 'c);
+type myType 'a 'b 'c = pri ('a, 'b, 'c);
```

Comment:
Do we have a test that proves that the following `Reason` code:

```rust
type private = (int, int);
```

Prints as ML:

```ocaml
type pri  = (int, int);
```